ci_framework/roles/env_op_images/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+          image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[0].imageID}');
+          echo -e "\n  OPENSTACK_IMG: ${image}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+
+    - name: Get openstack-operator image
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          pod_name=$(oc get pods -l openstack.org/operator-name=openstack --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}' );
+          image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[1].imageID}');
+          echo -e "\ncifmw_openstack_operator_image:\n  OPENSTACK_OP_IMG: ${image}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+
+    - name: Get Rabbitmq-operator image
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          pod_name=$(oc get pods -l app.kubernetes.io/component=rabbitmq-operator --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}');
+          image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[0].imageID}');
+          echo "  RABBITMQ_OP_IMG: ${image}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+
+    - name: Get rest of the operator images
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          for pod in $(oc get pod -l openstack.org/operator-name --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | grep -v 'openstack-operator-controller-manager' | awk '{print $1}');  do
+             operator_name=$(oc get pod $pod -o jsonpath='{.metadata.labels.openstack\.org/operator-name}' -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} --no-headers=true | tr '[:lower:]' '[:upper:]');
+             image_name=$(oc get pod $pod -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[1].image}' );
+             echo "  ${operator_name}_OP_IMG: ${image_name}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+          done
+
+    - name: Get openstack service images
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          echo -e "\ncifmw_openstack_svc_image:"  >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+          for pod in $(oc get pod  --no-headers=true  -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} | grep -v Completed | awk '{print $1}');  do
```

Comment:
@pablintino guess there's no cleaner way to get the data without nested loop? Care to have a look, you know more than I do :)